### PR TITLE
RCHAIN-3954: refactor integration test

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
https://rchain.atlassian.net/browse/RCHAIN-3954
refactor a little bit of integration test.
Use pyrchain instead of rnode command to control rnode 

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: integration-tests/test/rnode.py [hunk 1]
```diff
@@ -142,35 +163,49 @@ def get_peer_node_ip(self, network_name: str) -> str:
...
-        show_block_output = self.show_block(hash)
-        block_info = parse_show_block_output(show_block_output)
-        return block_info
-
-    def get_block(self, block_hash: str) -> str:
-        try:
-            return self.rnode_command('show-block', block_hash, stderr=False)
-        except NonZeroExitCodeError as e:
-            raise GetBlockError(command=e.command, exit_code=e.exit_code, output=e.output)
+    def show_block(self, hash: str) -> BlockInfo:
```
Comment (tgrospic) [inline integration-tests/test/rnode.py]: For me `get_block` sounds better as the same as API name.

**Review by tgrospic — APPROVED** [review]
Very good improvement!

I was expecting to see some speed up when running tests.

**Review by nzpr — APPROVED** [review]
LGTM

**Pull request merged**